mod/style/set/type/css.rb

```diff
@@ -1,2 +1,18 @@
 include_set Abstract::Css
-include_set Abstract::AssetInputter, input_format: :css, input_view: :compressed
+include_set Abstract::AssetInputter, input_format: :css, input_view: :core
+
+# require 'w3c_validators'
+require 'sass'
+
+event :validate_css, :validate, on: :save, changed: %i[type_id content] do
+  #validator = W3CValidators::CSSValidator.new
+  #results = validator.validate_text(content)
+  #results.errors.each do |error|
+  #  errors.add :content, "#{error.message} (line #{error.line})"
+  #end
+  begin
+    ::Sass::SCSS::Parser.new(content, nil, nil).parse
+  rescue ::Sass::SyntaxError => e
+    errors.add :content, e.message
+  end
```

Comment:
Css validation is harder than expected. 
Our Scss-Parser (SassC) can't validate plain css. The commented solution above (W3CValidator) needs a web request for the validation. The solution I picked uses a deprecated Scss-Parser.